compiler/rustc_ast_passes/src/ast_validation.rs

```diff
@@ -364,7 +364,11 @@ impl<'a> AstValidator<'a> {
         self.err_handler().emit_err(errors::BoundInContext { span, ctx });
     }
 
-    fn check_foreign_ty_genericless(&self, generics: &Generics, where_span: Span) {
+    fn check_foreign_ty_genericless(
+        &self,
+        generics: &Generics,
+        where_clauses: &(TyAliasWhereClause, TyAliasWhereClause),
```

Comment:
I think it should be `where_clause`, singular?

Reply:
Is it singular? Since there are two "where clause"s right...?

Reply:
nvm, I didn't notice there is a tuple ><

can you then split those, so it's `before_where_clause: TyAliasWhereClause, after_where_clause: TyAliasWhereClause`?

Reply:
Sure